ui/components/component-library/avatar-token/README.mdx

```diff
@@ -34,34 +34,87 @@ Defaults to `md`
   <Story id="ui-components-component-library-avatar-token-avatar-token-stories-js--size" />
 </Canvas>
 
-### Token Name
+```jsx
+import { AvatarToken } from '../../component-library';
 
-Use the `tokenName` prop to set the initial letter of the `AvatarToken`. This will be used as the fallback display if no image url is passed to the `tokenImageUrl` prop.
+  <AvatarToken size={SIZES.XS} />
+  <AvatarToken size={SIZES.SM} />
+  <AvatarToken size={SIZES.MD} />
+  <AvatarToken size={SIZES.LG} />
+  <AvatarToken size={SIZES.XL} />
+```
+
+### Name
+
+Use the `name` prop to set the initial letter of the `AvatarToken`. This will be used as the fallback display if no image url is passed to the `src` prop.
 
 <Canvas>
-  <Story id="ui-components-component-library-avatar-token-avatar-token-stories-js--token-name" />
+  <Story id="ui-components-component-library-avatar-token-avatar-token-stories-js--name" />
 </Canvas>
 
-### Token Image Url
+```jsx
+import { AvatarToken } from '../../component-library';
+
+  <AvatarToken name="A" />
+  <AvatarToken name="G" />
+```
 
-Use the `tokenImageUrl` prop to set the image to be rendered of the `AvatarToken`.
+### Src
+
+Use the `src` prop to set the image to be rendered of the `AvatarToken`.
 
 <Canvas>
-  <Story id="ui-components-component-library-avatar-token-avatar-token-stories-js--token-image-url" />
+  <Story id="ui-components-component-library-avatar-token-avatar-token-stories-js--src" />
 </Canvas>
 
+```jsx
+import { AvatarToken } from '../../component-library';
+
+  <AvatarToken src="./AST.png" />
+  <AvatarToken src="./BAT_icon.svg" />
+```
+
 ### Show Halo
 
-If we want to display the component with halo effect. Only works if an image url is supplied to `tokenImageUrl`
+If we want to display the component with halo effect. Only works if an image url is supplied to `src`
```

Comment:
I think in the description of the prop we want to specifically mention it like in `src` and `size`

```suggestion
Use the `showHalo` prop to display the component with halo effect. Only works if an image url is supplied to `src`
```